fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.g4

```diff
@@ -135,6 +136,7 @@ HLL_UNION: 'HLL_UNION';
 HOUR: 'HOUR';
 IDENTIFIED: 'IDENTIFIED';
 IF: 'IF';
+IMPERSONATE: 'IMPERSONATE';
```

Comment:
should make new added keywords nonReserve

Reply:
done